### PR TITLE
Add learning outcomes

### PR DESCRIPTION
**Pull request by joelostblom**
@SaraMati Following up on your comment in #1. I removed the following from the draft goals since they were not included this year:

- Focus on high dimensional data.
- Discriminate between interpretable and black box models and correctly setup validated statistical models
- Work in a productive and collaborative environment in a team-based project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 This course is offered by the Institute of Biomaterials and BiomedicalEngineering
 ([course descriptions page](https://ibbme.utoronto.ca/current-students/course-calendar/course-descriptions/))
 at the University of Toronto.
+It is intended for graduate students from different backgrounds
+who have entered the Neuromodulation/Neuroscience/Biomedical engineering program.
```
Comment (SaraMati) [inline README.md]:
```suggestion
who have entered any of the  Biomaterial, Biomedical or clinical engineering programs.
```

File: README.md [hunk 2]
```diff
@@ -3,6 +3,8 @@
 This course is offered by the Institute of Biomaterials and BiomedicalEngineering
```
Comment (SaraMati) [inline README.md]:
```suggestion
This course is offered by the Institute of Biomaterials and Biomedical Engineering
```

File: README.md [hunk 3]
```diff
@@ -18,6 +20,18 @@ The class will involve working through practical examples
 and solving real problems through a mix of online and live workshops.
```
Comment (SaraMati) [inline README.md]:
```suggestion
and solving scientific problems through a mix of online video lectures and live-coding sessions in which the students do the programming live with the instructor in the class.
```

File: README.md [hunk 4]
```diff
@@ -18,6 +20,18 @@ The class will involve working through practical examples
...
 
+# Learning outcomes
+
+Students will learn how to
+
+- Apply "tidy data" principles.
+- Develop a proficiency in coding and doing data analysis in Python.
+- Clean, manipulate, and manage data in Python.
+- Write well-documented and modular Python code.
+- Apply common statistical techniques to biomedical data.
```
Comment (SaraMati) [inline README.md]:
```suggestion
- Apply common statistical and basic machine learning techniques to biomedical data.
```

**Review by SaraMati — COMMENTED** [review]
some minor edits, apply if you see fit

**Comment by joelostblom** [comment]
Thanks @SaraMati !

**Pull request merged**